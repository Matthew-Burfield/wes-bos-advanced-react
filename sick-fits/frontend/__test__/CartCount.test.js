import CartCount from "../components/CartCount";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

describe("<CartCount />", () => {
  it("should render", () => {
    shallow(<CartCount />);
  });

  it("should match the snapshot", () => {
    const wrapper = shallow(<CartCount />);
    expect(toJSON(wrapper)).toMatchSnapshot();
    wrapper.setProps({ count: 1 });
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
