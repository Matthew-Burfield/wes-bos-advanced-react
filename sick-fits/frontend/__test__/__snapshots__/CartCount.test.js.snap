// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartCount /> should match the snapshot 1`] = `
<CartCount__AnimationStyles>
  <TransitionGroup
    childFactory={[Function]}
    component="div"
  >
    <CSSTransition
      className="count"
      classNames="count"
      key="0"
      timeout={
        Object {
          "enter": 400,
          "exit": 400,
        }
      }
      unmountOnExit={true}
    >
      <CartCount__Dot>
        0
      </CartCount__Dot>
    </CSSTransition>
  </TransitionGroup>
</CartCount__AnimationStyles>
`;

exports[`<CartCount /> should match the snapshot 2`] = `
<CartCount__AnimationStyles>
  <TransitionGroup
    childFactory={[Function]}
    component="div"
  >
    <CSSTransition
      className="count"
      classNames="count"
      key="1"
      timeout={
        Object {
          "enter": 400,
          "exit": 400,
        }
      }
      unmountOnExit={true}
    >
      <CartCount__Dot>
        1
      </CartCount__Dot>
    </CSSTransition>
  </TransitionGroup>
</CartCount__AnimationStyles>
`;
